### ISSUE TITLE
Getting better results on yolov5

### ISSUE THREAD

**Issue by bezir** [ultralytics/ultralytics#1587]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I did not change any part of the code or hyperparameters. Trained on the exact same datasets and YOLOv5 is slightly better. I cannot really figure it out the reason.

**YOLOv5**
![results](https://user-images.githubusercontent.com/66210642/232033528-fb0cdaa6-b64e-469e-a86c-337a942a153c.png)
<img width="257" alt="v5-parameters" src="https://user-images.githubusercontent.com/66210642/227346869-e01802d9-854f-4543-b0fb-d6cdffdcebab.png">

**YOLOv8**
![resultsv8](https://user-images.githubusercontent.com/66210642/232033581-fb00976d-872d-4340-89b4-95f3d78fb827.png)
<img width="157" alt="v8-paramters" src="https://user-images.githubusercontent.com/66210642/227347446-98050fa7-3887-40d4-a0f7-056c68232f7c.png">

### Additional

Labels:
Avarage: 2k images per class. (54 class)
![labels](https://user-images.githubusercontent.com/66210642/232033803-fbd5cf87-045b-41b9-bda7-2edba683473e.jpg)

Hyperparameters:
[v5-args.yaml.zip](https://github.com/ultralytics/ultralytics/files/11055776/v5-args.yaml.zip)
[v8-args.yaml.zip](https://github.com/ultralytics/ultralytics/files/11055781/v8-args.yaml.zip)

**Comment by glenn-jocher** [comment]
@abdullahbzr there, I've fixed the title for you. Your YOLOv8 results are better than your YOLOv5 results, higher mAP50-95.

**Comment by abdullahbzr** [comment]
@glenn-jocher It appears that my interpretation of the case may have been biased, as I was searching for a justification for my poor results when training with a larger number of classes. I may have allowed my preconceptions to cloud my judgment. Nevertheless, I will strive to thoroughly investigate the situation and either construct a sound argument or re-run my tests to obtain more precise and reliable results.

**Comment by abdullahbzr** [comment]
@glenn-jocher It appears that my interpretation of the case may have been biased, as I was searching for a justification for my poor results when training with a larger number of classes. I may have allowed my preconceptions to cloud my judgment. Nevertheless, I will strive to thoroughly investigate the situation and either construct a sound argument or re-run my tests to obtain more precise and reliable results.

**Comment by glenn-jocher** [comment]
There's no absolutes here, but in general you should expect improved performance with v8 than with v5 across most domains and tasks.

**Comment by bezir** [comment]
54 class, average 2500 labels for each class.

**YOLOv8**
![resultsv8](https://user-images.githubusercontent.com/66210642/232032988-e8990336-9e1b-49ba-b309-5b687ca58d77.png)

**YOLOv5**
![results](https://user-images.githubusercontent.com/66210642/232032982-6a15de06-ca05-471d-bb05-43386364bc5f.png)

@glenn-jocher what would your comment be?

**Comment by bezir** [comment]
I would really appreciate any comment on this.

**Comment by glenn-jocher** [comment]
@bezir, since you have provided the same number of epochs, the same learning rate schedule and the same augmentation procedure for both YOLOv5 and YOLOv8, the difference in performance is unlikely to be due to the factors you have controlled for. Nevertheless, it is always important to benchmark your models with adequate validation experiments, so as to ensure that observed differences are not merely chance occurrences. Additionally, it would be difficult to draw general conclusions from these results without access to more data and evaluation metrics.
